lib/remote/jsonrpcconnection.cpp

```diff
@@ -78,9 +80,36 @@ void JsonRpcConnection::HandleIncomingMessages(boost::asio::yield_context yc)
 		m_Seen = Utility::GetTime();
 
 		try {
+			auto start (ch::steady_clock::now());
+			ch::steady_clock::duration cpuBoundDuration;
+
+			String rpcMethod;
+
+			Log statsLog(LogDebug, "JsonRpcConnection");
+
+			Defer addLogStats ([this, &statsLog, &rpcMethod,  &start, &cpuBoundDuration]() {
+				statsLog << "Processing JSON-RPC '" << rpcMethod << "' message for identity '" << m_Identity << "'";
+
+				if (ch::duration_cast<ch::seconds>(cpuBoundDuration).count()) {
```

Comment:
```suggestion
				if (cpuBoundDuration >= ch::seconds(1)) {
```

Just use the same type of check like a few lines below, it's much more readable.